packages/wrangler/src/dev.tsx

```diff
@@ -505,9 +554,13 @@ export async function startDev(args: ArgumentsCamelCase<DevArgs>) {
 					bindings={bindings}
 					crons={config.triggers.crons}
 					logLevel={args.logLevel}
+					logPrefix={args.logPrefix}
 					onReady={args.onReady}
 					inspect={args.inspect ?? true}
 					showInteractiveDevSession={args.showInteractiveDevSession}
+					forceLocal={args.forceLocal}
```

Comment:
what is forceLocal, and why can't we use initialMode here?

Reply:
`forceLocal` prevents users from ever switching to remote mode. It disables the hotkey and hides it from the UI.